src/pages/components/content-switcher/style.mdx

```diff
@@ -68,70 +111,86 @@ Content switcher label text should be set in sentence case, with only the first
 
 ## Structure
 
 Content switchers must have at least two options for the user to choose from.
-Each container that makes up the content switcher is equal in size. The width of
-a container is determined by the length of the longest container option text
-plus the 16 px / 1rem on both sides of the text.
+Each container that makes up the content switcher is equal in size.
+
+| Element    | Property                    | px / rem  | Spacing token |
+| ---------- | --------------------------- | --------- | ------------- |
+| Container  | corner radius               | 4px       | –             |
+| Label text | padding-left, padding-right | 16 / 1    | `$spacing-05` |
+| Icon (sm)  | padding-left, padding-right | 8 / 0.5   | `$spacing-03` |
+| Icon (md)  | padding-left, padding-right | 12 / 0.75 | `$spacing-04` |
+| Icon (lg)  | padding-left, padding-right | 14/ 0.875 | –             |
+| Divider    | border                      | 1px       | –             |
 
-| Element      | Property                    | px / rem | Spacing token |
-| ------------ | --------------------------- | -------- | ------------- |
-| Container    | corner radius               | 4px      | –             |
-| Divider      | border                      | 1px      | –             |
-| Button label | padding-left, padding-right | 16 / 1   | `$spacing-05` |
+### Text content switcher
+
+The width of a text container is determined by the length of the longest label
+text within its content switcher.
 
 <div className="image--fixed">
 
-![Content switcher structure and spacing measurements](images/content-switcher-style-1.png)
+![Text content switcher structure and spacing measurements](images/content-switcher-style-1.png)
 
 </div>
 
 <Caption>
-  Structure and spacing measurements for the default size content switcher | px
-  / rem
+  Structure and spacing measurements for the text content switcher | px / rem
 </Caption>
 
-### Icon variant
+### Icon content switcher
 
-| Element  | Property      | px / rem | Spacing token |
-| -------- | ------------- | -------- | ------------- |
-| Tab (sm) | height, width | 32 / 2   | –             |
-|          | svg           | 16 x 16  | –             |
-|          | padding       | centered | –             |
-| Tab (md) | height, width | 40 / 2.5 | –             |
-|          | svg           | 16 x 16  | –             |
-|          | padding       | centered | –             |
-| Tab (lg) | height, width | 48 / 3   | –             |
-|          | svg           | 20 x 20  | –             |
-|          | padding       | centered | –             |
+The width of an icon container is determined by the fixed size within its
+content switcher.
 
 <div className="image--fixed">
 
-![Structure and spacing measurements for icon content switcher in px and rem](images/content-switcher-style-4.png)
+![Icon content switcher structure and spacing measurements](images/content-switcher-style-structure-icon.png)
```

Comment:
This image is not showing up correctly. I would try to export it again at 736px width and see if it corrects itself.

Reply:
Also I don't think we put a space between the 8px/.5rem